### ISSUE TITLE
Chat functionality

### ISSUE THREAD

**Issue by somu-code** [100XEnginners/CampusGhost#8]
Users should be able to chat with each other.

**Comment by Praashh** [comment]
is it personal chat or else can you elaborate @somu-code

**Comment by somu-code** [comment]
> is it personal chat or else can you elaborate @somu-code

Yes, it is a personal chat. The chat feature will have end-to-end encryption like we have in WhatsApp. Later we can work on chat rooms and all.
